src/cplus_plugin/conf.py

```diff
@@ -483,6 +489,7 @@ def save_priority_layer(self, priority_layer):
             settings.setValue("description", priority_layer["description"])
             settings.setValue("path", priority_layer["path"])
             settings.setValue("selected", priority_layer.get("selected", False))
+            settings.setValue("user_defined", priority_layer.get("user_defined", True))
```

Comment:
When initializing the plugin priority weighting layers for the first time(when installing the plugin) the `user_defined` will have a `True` value because it doesn't exist in the default priority layers dict!

Reply:
On initialization, I explicitly set `user_defined` to `False`:

![image](https://github.com/kartoza/cplus-plugin/assets/3840267/7897c80d-c6e1-449f-a7ee-1ed20dcce007)